src/screens/LinkContextScreen/elements/ContentRelated.js

```diff
@@ -28,12 +28,14 @@ const styles = StyleSheet.create({
     lineHeight: 24,
     paddingHorizontal: 20,
     marginTop: 12,
-    marginBottom: 12
+    marginBottom: 12,
+    color: COLORS.white2
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of the text has been changed to `COLORS.white2`. Ensure that this color change is consistent with the overall theme and accessibility standards. High contrast between text and background is important for readability.



<!-- This is an auto-generated comment by OSS CodeRabbit -->